capstone/src/main/java/com/google/sps/data/Classroom.java

```diff
@@ -1,94 +1,100 @@
 package com.google.sps.data;
-import java.io.*;
-import com.google.sps.data.Teacher;
-import com.google.sps.data.Student;
-import com.google.sps.data.Form;
+
+import com.google.appengine.api.datastore.Key;
+import com.google.appengine.api.datastore.Entity;
 import java.util.ArrayList;
 import java.util.List;
-import java.util.Random; 
-import com.google.appengine.api.datastore.Entity;
+import com.google.sps.service.DatabaseService;
 
 public class Classroom {
+    protected static final String CLASSROOM = "Classroom";
+    protected static final String TEACHER = "teacher";
+    protected static final String SUBJECT = "subject";
+    protected static final String STUDENTS = "students";
+    protected static final String FORMS = "forms";
 
-    private Teacher teacher;
-    private List<Student> students = new ArrayList<>();
-    private List<Form> forms = new ArrayList<>();
-    private String subject;
-    private Key key;
-
-    public Classroom(Teacher teacher, List<Student> students, List<Form> forms, String subject, Key key) {
-        this.teacher = teacher;
-        this.students = students;
-        this.forms = forms;
-        this.subject = subject;
-        this.key = key;
-    }
+    protected Entity entity;
 
-    // Setters
-    public void setTeacher(Teacher teacher) {
-        this.teacher = teacher;
+    public Classroom(Entity entity) {
+        this.entity = entity;
     }
 
-    public void setSubject(String subject) {
-        this.subject = subject;
+    public Classroom(User teacher, String subject) { 
+        this.entity = new Entity(CLASSROOM);
```

Comment:
Just calling it out, no change needed. Saying `CLASSROOM` and `Classroom.CLASSROOM' while in the context of the `Classroom` class are synonymous.

Reply:
got it

Reply:
I will request a change though. When using static variables, you should always preface it with the classname to disambiguate it from instance variables.

Reply:
sounds good